nibabel/nifti2.py

```diff
@@ -239,6 +240,31 @@ class Nifti2Image(Nifti1Image):
     """
     header_class = Nifti2Header
 
+    @property
+    def is_cifti(self):
```

Comment:
I'd rather see this as part of `guessed_image_type` pathway, rather than as a property on the unrelated (in the object hierarchy) `Nifti2Image`. It's not used anywhere else, this is the kind of stuff found in that pathway already.